AssetsPickerViewController/Classes/Assets/AssetsManager.swift

```diff
@@ -482,15 +492,22 @@ extension AssetsManager {
         if isRefetch {
             assetArray.removeAll()
         }
-        
-        // set default album
-        select(album: defaultAlbum ?? cameraRollAlbum)
-        
+
+        // A crash happens because cameraRollAlbum is nil when it is force unwrapped here. This is a temporary workaround for the crash.
+        let targetAlbum: PHAssetCollection? = defaultAlbum ?? cameraRollAlbum
```

Comment:
The type of `cameraRollAlbum` can also be changed. It shouldn't be `PHAssetCollection!` anymore because it's clear that it CAN be `nil`. Changing the type to `PHAssetCollection?` will also force us to fix other places that might have the same issue.